django/core/checks/security/base.py

```diff
@@ -182,3 +187,9 @@ def check_xframe_deny(app_configs, **kwargs):
         settings.X_FRAME_OPTIONS == 'DENY'
     )
     return [] if passed_check else [W019]
+
+
+@register(Tags.security, deploy=True)
+def check_allowed_hosts(app_configs, **kwargs):
+    passed_check = not settings.ALLOWED_HOSTS
```

Comment:
@coldmind With the current code, `passed_check` will be `True` if `settings.ALLOWED_HOSTS` IS empty. That is backwards. `passed_check` should be `True` if `settings.ALLOWED_HOSTS` is NOT empty.